{
  "artifacts": [],
  "command": "train",
  "experiment": {
    "base_dir": "/home/katy/imitation/src/imitation/scripts/config",
    "dependencies": [
      "imitation==0.2.0",
      "numpy==1.21.0",
      "sacred==0.8.2"
    ],
    "mainfile": "train_adversarial.py",
    "name": "train_adversarial",
    "repositories": [
      {
        "commit": "740f4eefd6eb6aa04e5fee68151336dc1861615d",
        "dirty": true,
        "url": "http://github.com/HumanCompatibleAI/imitation"
      }
    ],
    "sources": [
      [
        "train_adversarial.py",
        "_sources/train_adversarial_cec6a35f0f6f6fe2010eb5fb8d6d8127.py"
      ]
    ]
  },
  "fail_trace": [
    "Traceback (most recent call last):\n",
    "  File \"/home/katy/miniconda3/envs/HumanCompatibleAI/lib/python3.7/site-packages/gym/envs/registration.py\", line 121, in spec\n    return self.env_specs[id]\n",
    "KeyError: 'CliffWalker-v0'\n",
    "\nDuring handling of the above exception, another exception occurred:\n\n",
    "Traceback (most recent call last):\n",
    "  File \"/home/katy/miniconda3/envs/HumanCompatibleAI/lib/python3.7/site-packages/sacred/config/captured_function.py\", line 42, in captured_function\n    result = wrapped(*args, **kwargs)\n",
    "  File \"/home/katy/imitation/src/imitation/scripts/train_adversarial.py\", line 172, in train\n    max_episode_steps=max_episode_steps,\n",
    "  File \"/home/katy/imitation/src/imitation/util/util.py\", line 65, in make_vec_env\n    spec = gym.spec(env_name)\n",
    "  File \"/home/katy/miniconda3/envs/HumanCompatibleAI/lib/python3.7/site-packages/gym/envs/registration.py\", line 148, in spec\n    return registry.spec(id)\n",
    "  File \"/home/katy/miniconda3/envs/HumanCompatibleAI/lib/python3.7/site-packages/gym/envs/registration.py\", line 131, in spec\n    raise error.UnregisteredEnv('No registered env with id: {}'.format(id))\n",
    "gym.error.UnregisteredEnv: No registered env with id: CliffWalker-v0\n"
  ],
  "heartbeat": "2021-07-26T19:11:38.420310",
  "host": {
    "ENV": {},
    "cpu": "AMD Ryzen 5 3600 6-Core Processor",
    "gpus": {
      "driver_version": "470.57.02",
      "gpus": [
        {
          "model": "NVIDIA GeForce GTX 1070",
          "persistence_mode": false,
          "total_memory": 8116
        }
      ]
    },
    "hostname": "katy-desktop",
    "os": [
      "Linux",
      "Linux-5.4.0-80-generic-x86_64-with-debian-buster-sid"
    ],
    "python_version": "3.7.10"
  },
  "meta": {
    "command": "train",
    "options": {
      "--beat-interval": null,
      "--capture": null,
      "--comment": null,
      "--debug": false,
      "--enforce_clean": false,
      "--file_storage": null,
      "--force": false,
      "--help": false,
      "--loglevel": null,
      "--mongo_db": null,
      "--name": null,
      "--pdb": false,
      "--print-config": false,
      "--priority": null,
      "--queue": false,
      "--s3": null,
      "--sql": null,
      "--tiny_db": null,
      "--unobserved": false,
      "COMMAND": null,
      "UPDATE": [],
      "help": false,
      "with": false
    }
  },
  "resources": [],
  "result": null,
  "start_time": "2021-07-26T19:11:38.352806",
  "status": "FAILED",
  "stop_time": "2021-07-26T19:11:38.422428"
}